tests/general/LicenseDetectionBearTest.py

```diff
@@ -0,0 +1,66 @@
+from queue import Queue
+import os
+
+from bears.general.LicenseDetectionBear import LicenseDetectionBear
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
+from coalib.testing.BearTestHelper import generate_skip_decorator
+from coalib.results.Result import Result
+from coalib.settings.Section import Section
+
+
+def get_testfile_path(name):
+    return os.path.join(os.path.dirname(__file__),
+                        'licensedetect_test_files',
+                        name)
+
+
+def load_testfile(name):
+    with open(get_testfile_path(name)) as fl:
+        contents = fl.read()
+
+    return contents
+
+
+@generate_skip_decorator(LicenseDetectionBear)
+class LicenseDetectionBearTest(LocalBearTestHelper):
+
+    def setUp(self):
+        self.uut = LicenseDetectionBear(Section('name'), Queue())
+
+    def test_license(self):
+        test_file_path = get_testfile_path('apache_license_2.py')
+        file_contents = [load_testfile(test_file_path)]
+        self.check_results(
+            self.uut,
+            file_contents,
+            [Result.from_values('LicenseDetectionBear',
+                                test_file_path + ': Apache (v2.0)',
+                                file=test_file_path)],
+            filename=test_file_path)
+
+    def test_copyright(self):
+        test_file_path = get_testfile_path('mit_license.py')
+        file_contents = [load_testfile(test_file_path)]
+        self.check_results(
+            self.uut,
+            file_contents,
+            [Result.from_values('LicenseDetectionBear',
+                                test_file_path + ': MIT/X11 (BSD like)',
+                                file=test_file_path),
+             Result.from_values('LicenseDetectionBear',
+                                '[Copyright: 2008 <name of author>]',
```

Comment:
IIUC, there is not way this can be achieved since there is no option to specify a regex that the copyright can match unlike https://pypi.python.org/pypi/flake8-copyright.